moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -906,6 +906,11 @@ class MoveGroupInterface
       This removes any path constraints set in previous calls to setPathConstraints(). */
   void clearPathConstraints();
 
+  moveit_msgs::TrajectoryConstraints getTrajectoryConstraints() const;
+  bool setTrajectoryConstraints(const std::string& constraints);
```

Comment:
This variant doesn't seem to have an implementation. I think, it's superfluous.

Reply:
Nice catch! Removed